modules/dnn/src/layers/mvn_layer.cpp

```diff
@@ -118,12 +118,10 @@ class MVNLayerImpl CV_FINAL : public MVNLayer
 
     virtual bool supportBackend(int backendId) CV_OVERRIDE
     {
-#ifdef HAVE_INF_ENGINE
-        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 || backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
+        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019)
```

Comment:
Please keep `#ifdef` (make happy code coverage tools).

Use separate `if`s for backends (avoid overcomplicated conditions which are hard to merge):
```
else if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH)
    return true;
```

Reply:
Thanks! Replaced